.github/workflows/validate-codeowners.yml

```diff
@@ -1,7 +1,5 @@
 name: Validate Codeowners
 on:
-  workflow_dispatch:
-  
```

Comment:
You probably did not mean to delete this, but in any case, the `workflow_dispatch` trigger needs to remain